src/test/java/com/google/javascript/clutz/partialCrossModuleTypeImports/goog_legacy_namespace_exporter.js

```diff
@@ -0,0 +1,11 @@
+goog.module("goog.legacy.namespace.exporter");
+goog.module.declareLegacyNamespace();

```

Comment:
add a comment about the role of declareLegacyNamespace in the test. Would this test work if this was just a goog.provide('goog.legacy.namespace.exporter')?

Reply:
Added a comment explaining that the declare legacy namespace isn't load bearing, it's just the example I came across.